pyvista/plotting/render_window_interactor.py

```diff
@@ -780,6 +781,29 @@ def enable_rubber_band_2d_style(self):
         self._style_class = None
         self.update_style()
 
+    def enable_user_style(self):
+        """Set the interactive style to user.
+
+        This interaction style allows you to customize interaction events. By
+        default, this interaction style does nothing.
```

Comment:
This sounds useful. But it's not at all clear to me how events can be customized. Shouldn't we mention something about this, maybe just link to other parts of the documentation if it's already something that exists? If this is completely novel, I think we should consider adding another docstring example that shows how to configure a single event.

Reply:
Good point. I mostly added this here just to disable user events to enable a fancy geometry manipulation widget I've been working on for a separate application:
![tmp](https://github.com/pyvista/pyvista/assets/11981631/511cb1c7-c70d-4893-a931-c8fed360116d)

I'll add the ability to customize at least one event.

Reply:
@tkoyama010 when @akaszynski said
> I'll add the ability to customize at least one event.

I understood that as a plan to update this PR with that feature. I'd prefer to wait until @akaszynski resolves this thread himself to avoid any misunderstandings.